arviz/plots/bpvplot.py

```diff
@@ -44,50 +44,50 @@ def plot_bpv(
     data : InferenceData
         :class:`arviz.InferenceData` object containing the observed and
         posterior/prior predictive data.
-    kind : str, default "u_value"
-        Type of plot to display ("p_value", "u_value", "t_stat").
-        For "p_value" we compute p := p(y* ≤ y | y). This is the probability of the data y being
-        larger or equal than the predicted data y*. The ideal value is 0.5 (half the predictions
-        below and half above the data).
-        For "u_value" we compute pi := p(yi* ≤ yi | y). i.e. like a p_value but per observation yi.
-        This is also known as marginal p_value. The ideal distribution is uniform. This is similar
-        to the LOO-pit calculation/plot, the difference is than in LOO-pit plot we compute
-        pi = p(yi* r ≤ yi | y-i ), where y-i, is all other data except yi.
-        For "t_stat" we compute := p(T(y)* ≤ T(y) | y) where T is any test statistic. See t_stat
-        argument below for details of available options.
-    t_stat : str, float, or callable, default "median"
-        Test statistics to compute from the observations and predictive distributions.
-        Allowed strings are "mean", "median" or "std".
-        Alternative a quantile can be passed as a float (or str) in the
-        interval (0, 1). Finally a user defined function is also
+    kind : {"p_value", "u_value", "t_stat"}, default "u_value"
+        Specify the kind of plot:
+
+           * The ``kind="p_value"`` computes :math:`p := p(y* \leq y | y)`. This is the probability of the 
+             data y being larger or equal than the predicted data y*. The ideal value is 0.5 
+             (half the predictions below and half above the data).
+           * The ``kind="u_value"`` argument computes :math:`p_i := p(y_i* \leq y_i | y)`. i.e. like a p_value 
+             but per observation :math:`y_i`. This is also known as marginal p_value. The ideal distribution 
+             is uniform. This is similar to the LOO-pit calculation/plot, the difference is than in 
+             LOO-pit plot we compute :math:`pi = p(y_i* r \leq y_i | y_{-i} )`, where :math:`y_{-i}`, is all 
+             other data except :math:`y_i`.
+           * The ``kind="t_stat"`` argument computes :math:`:= p(T(y)* \leq T(y) | y)` where T is any test statistic. 
+             See ``t_stat`` argument below for details of available options.
+             
+    t_stat : {"mean", "median", "std"},  default "median"
```

Comment:
This needs to be kept as before as this is now incorrect. Allowed strings are only mean, median or std, but it is not necessarily a string, it can also be a float or a user defined function (`callable` type in python)